### PR TITLE
Using cocoapod for native template

### PR DESCRIPTION
**Pull request by wmathurin**
For this pull request, I didn't copy over the updated template and createApp.sh
Once the changes on forceios.js are reviewed, we should move it to the SalesforceMobileSDK-iOS repo and stop using this repo.

### CODE REVIEW

File: forceios.js [hunk 1]
```diff
@@ -212,6 +201,11 @@ function updateApp(config) {
         process.exit(4);
     }
 
+    //
```
Comment (wmathurin) [inline forceios.js]: Update need to call out to pod update for native app and cordova for hybrid app
In its current form it wasn't working for hybrid applications

File: .gitmodules [hunk 2]
```diff
@@ -1,6 +1,3 @@
-[submodule "Dependencies"]
```
Comment (wmathurin) [inline .gitmodules]: We don't need the binaries anymore.
And therefore we probably don't need the SalesforceMobileSDK-iOS-Distribution repo anymore also?

**Comment by kchitalia** [comment]
:+1:

File: forceios.js (hunk 1 above)
Comment (bhariharan) [inline forceios.js]: So we're getting ```Cocoapods``` for hybrid as well with ```Cordova 4.0```?

File: .gitmodules (hunk 2 above)
Comment (bhariharan) [inline .gitmodules]: Don't we need this for hybrid?

File: forceios.js (hunk 1 above)
Comment (wmathurin) [inline forceios.js]: No hybrid app are using the cordova plugin.

File: .gitmodules (hunk 2 above)
Comment (wmathurin) [inline .gitmodules]: No they are using the cordova plugin which has all the static libraries etc.

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**